packages/logging/src/api/transport.js

```diff
@@ -22,13 +22,6 @@ export function sendReport(payload, transportOpts, attempt = 0) {
 	return transformedPayload;
 }
 
-function defaultPayloadMiddleware(payload) {
-	return {
-		time: new Date(),
-		...payload,
-	};
-}
-
 const defaultHandlers = {
 	success: response => {
 		console.info('Logging: reported', response);
```

Comment:
do we need to console info outside the dev environnement ?

Reply:
No, and in fact I don't think it's really useful even in dev mode.
I just didn't see that, let's remove it.